spring-xsuaa/pom.xml

```diff
@@ -49,6 +50,11 @@
 			<artifactId>caffeine</artifactId>
 			<version>${caffeine.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>io.projectreactor</groupId>
+			<artifactId>reactor-core</artifactId>
+			<version>3.2.8.RELEASE</version>
```

Comment:
```suggestion
			<version>${reactor.version}</version>
```

Reply:
Define the version below. Unfortunately I can't suggest changes for non-changed parts of the code.